nmmo/core/env.py

```diff
@@ -41,15 +40,7 @@ def __init__(self,
 
     self._gamestate_generator = GameStateGenerator(self.realm, self.config)
     self.game_state = None
-    # Default task: rewards 1 each turn agent is alive
-    self.tasks: List[Tuple[Task,float]] = None
-    self._task_encoding = None
-    self._task_embedding_size = -1
-    t = default_task(self.possible_agents)
-    self.change_task(t,
-                     embedding_size=self._task_embedding_size,
-                     task_encoding=self._task_encoding,
-                     reset=False)
+    self.tasks = nmmo_default_task(self.possible_agents)
```

Comment:
could you do task_api.make_nmmo_default_tasks()

Reply:
explicitly stated where the default task comes from.